### DISCUSSION TITLE
Backup - allow multiple instances

### DISCUSSION THREAD

**Discussion by mrx23dot** [Authenticator-Extension/Authenticator#1263] [Ideas]
Lets say I want to backup 

- workplace browser and 
- home browser 

to same Drive, in that case they would override each other if it happened on same day, so could we store the name of the PC too in filename?

**Comment by mymindstorm** [comment]
It should create two separate folders with the same name if you have two different copies of the extension. You should be able to rename both folders in google drive without issues. Try doing a manual backup after renaming a folder to test this.
